### DISCUSSION TITLE
Theory: all this fuzz about Stallman is attempt of BigTech to get reed off Stallman, because he teaches people how to be free.

### DISCUSSION THREAD

**Discussion by AnimusPEXUS** [rms-support-letter/rms-support-letter.github.io#3590] [General]
how do you think?

**Comment by pda0** [comment]
This is almost the second most popular theory in the Russian-speaking segment of the Internet. It argues that this is part of an attempt by corporations to take control over large open source projects by displacing current leaders and promoting the "right" people in their place. Welcome to cyberpunk. Kinda... 🙄

**Comment by nukeop** [comment]
Stallman has been fighting for software freedom, and your freedom, for over 40 years.
Removing him for frivolous reasons would be very beneficial to some corporations.

**Reply by 6r1d** [reply]
I agree with you there

**Comment by waltercool** [comment]
Most of the people who signed the letter don't believe really in free software, many of the companies supporting this coup hate the GPL restrictions because it's bad for business, reason why most companies use BSD instead.

Some individuals who signed the letter support non-free licenses like "hippocratic license", which is GPL except they can revoke you permission when they want.

But the worst part, they don't believe in democracy. Board members are elected by votes, and Stallman was elected by votes. The sole reason the anti-FSF letter exist, it's because those people are just an angry mob who doesn't vote for FSF.

**Comment by franpoli** [comment]
The tech giants are going too far by letting their employees interfere on their behalf in the decision-making of free organizations for purposes diametrically opposed to their objectives.

```
commit: 1791d4a, date: 2021-03-26, message: "Add myself", author: Jo Shields, email: j######d@microsoft.com
commit: b4a0c80, date: 2021-03-26, message: "Add Mark Hansen's name", author: Mark Hansen, email: m########n@google.com
commit: 1a81683, date: 2021-03-25, message: "Signing.", author: Stephen Canon, email: s####n@apple.com
commit: 75bb8e2, date: 2021-03-25, message: "Add myself", author: Stephen Hemminger, email: s######n@microsoft.com
commit: 1a6f332, date: 2021-03-25, message: "Signing", author: Julia Ferraioli, email: j############i@google.com
commit: 514ef21, date: 2021-03-25, message: "Update index.md", author: Chris Blume, email: c####e@google.com
commit: acf3408, date: 2021-03-24, message: "Add Holden Karau (Apache Spark)", author: Holden Karau, email: h####u@apple.com
commit: 725ddf1, date: 2021-03-24, message: "Add Jaana Dogan", author: JBD, email: j#d@amazon.com
commit: c5f8e14, date: 2021-03-24, message: "Signing matters", author: Jim Perrin, email: J########n@microsoft.com
```
The people above are respectless to the FSF community, to their own colleagues who might not be sharing their biased opinions and lastly to their own coorporation if they act in their personal name. Have we heard anything from Microsoft, Google, Amazon and Apple supporting these persons in their action?
